app/views/diary_entry/view.html.erb

```diff
@@ -10,7 +10,11 @@
 
 <a id="comments"></a>
 <div class='comments'>
-<%= render :partial => 'diary_comment', :collection => @entry.visible_comments %>
+  <% if @reported_comment %>
+    <%= render :partial => 'diary_comment', :collection => @reported_comment %>
```

Comment:
Is there a separate state for displaying only reported comments? Can it be that some users will see only one reported comment instead of a full comment list?